federatedscope/contrib/data/example.py

```diff
@@ -1,7 +1,7 @@
 from federatedscope.register import register_data
 
 
-def MyData(config):
+def MyData(config, client_cfgs):
```

Comment:
Shall we provide `client_cfgs` in `MyData`?

Reply:
I believe the `client_cfgs ` will useful in `MyData` when using personalized `cfg`. (`cfg.dataloader.batch_size` varies.)

Reply:
So we can set `client_cfgs=None` by default here since it just serves as an example.

Reply:
Agree!